packages/combobox/src/Combobox.ts

```diff
@@ -211,6 +214,7 @@ export class Combobox extends Textfield {
             return;
         }
         this.value = this.activeDescendant.itemText;
+        this.handleChange();
```

Comment:
It looks like `selectDescendant()` is _only_ used when the `Enter` key is pressed. I wonder if we can get the same, but slightly better, simply by `this.activeDescendant.click()`. In theory, this would trigger the lifecycle of the `<sp-menu>` and hit all of the appropriate `change`, etc. pathways ensuing all selections were "the same".

Reply:
`this.activeDescendant.click()` does not really work, since it is defined as `MenuItem | ComboboxOption`, but what can be done and works is similar to what's inside `scrollToActiveDescendant`, but instead of scroll you click.